samples/04_gis_analysts_data_scientists/visualize_monthly_changes_in_hirakund_reservoir_using_video.ipynb

```diff
@@ -60,25 +60,1240 @@
    "cell_type": "code",
```

Comment:
<p>Is there a particular reason you stopped execution of this cell?</p><p> </p><p>If you don't need to install, leave the cell as is without running it, and add a markdown note above saying that 'If you do not have imageio you may install it as follows, else proceed to the cell following that'.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1747/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>thanks for catching this, corrected</p>